test/rebar_install_deps_SUITE.erl

```diff
@@ -199,39 +199,22 @@ mdeps(m_circular2) ->
     {[{"B", [{"c", [{"b", []}]}]}],
      [],
      {error, {rebar_prv_install_deps, {cycles, [[<<"B">>,<<"C">>]]}}}};
-mdeps(m_circular3) ->
-    %% Spot the circular dep due to being to low in the deps tree
-    %% but as a source dep, taking precedence over packages
-    {[{"B", [{"C", "2", [{"B", []}]}]},
-      {"c", "1", [{"d",[]}]}],
-     [],
-     {error, {rebar_prv_install_deps, {cycles, [[<<"B">>,<<"C">>]]}}}};
 mdeps(m_pick_source1) ->
     {[{"B", [{"D", []}]},
       {"c", [{"d", []}]}],
      ["d"],
      {ok, ["B", "c", "D"]}};
 mdeps(m_pick_source2) ->
-    {[{"b", [{"d", []}]},
-      {"C", [{"D", []}]}],
-     ["d"],
-     {ok, ["b", "C", "D"]}};
-mdeps(m_pick_source3) ->
```

Comment:
We could have kept this one and instead changed the warning for `D`, and the final deps as `["b", "C", "d"]`